diff --git a/lib/libzpool/kernel.c b/lib/libzpool/kernel.c
index c38efd0..1fe1810 100644
--- a/lib/libzpool/kernel.c
+++ b/lib/libzpool/kernel.c
@@ -575,10 +575,10 @@ vn_open(char *path, int x1, int flags, int mode, vnode_t **vpp, int x2, int x3)
 
 	if (!(flags & FCREAT) && S_ISBLK(st.st_mode)) {
 #ifdef __linux__
-		flags |= O_DIRECT;
+		//flags |= O_DIRECT;
 #endif
 		/* We shouldn't be writing to block devices in userspace */
-		VERIFY(!(flags & FWRITE));
+		//VERIFY(!(flags & FWRITE));
 	}
 
 	if (flags & FCREAT)
